airflow/www/static/js/components/RenderedJsonField.tsx

```diff
@@ -47,7 +47,7 @@ const JsonParse = (content: string | object) => {
       contentJson = content;
     }
     contentFormatted = JSON.stringify(contentJson, null, 4);
-    isJson = true;
+    isJson = Number.isNaN(contentJson); // ensure numbers/bool are not treated as JSON
```

Comment:
I think this actually returns `false` for everything that’s actually JSON…

```js
var c = JSON.stringify({a: 1})
console.log(Number.isNaN(c))  // false
```

Reply:
Uuups. Yes. Reading docs helps. Still I am wondering why actually it _was_ working with this.

Anyway, how about as I pushed a different approach? After parsing as JSON should be an object (which is true for Array+Dict)